### PR TITLE
Finishing autofix

### PR DESCRIPTION
**Pull request by evgeniy-r**
This PR fixes the issue #285.
There is a little duplication, but I think that is OK for now.

As far as I can see, the autofix feature is finally completed.

#### ✔ Checklist:

- [x] This PR has been added to [CHANGELOG.md](https://github.com/dotenv-linter/dotenv-linter/blob/master/CHANGELOG.md) (at the top of the list);
- [x] Tests for the changes have been added (for bug fixes / features).

### CODE REVIEW

**Comment by evgeniy-r** [comment]
@mgrachev, I remember you mentioned that we should remove the `Fixed/Unfixed` feature.

I can do this, it will simplify the code.

**Comment by mgrachev** [comment]
> @mgrachev, I remember you mentioned that we should remove the `Fixed/Unfixed` feature.
> 
> 
> 
> I can do this, it will simplify the code.

I have already done it, but haven't published yet.

**Review by mgrachev — COMMENTED** [review]
@evgeniy-r Thank you for your contribution! 👍

I've left some comments. Please look at them 👀

File: src/fixes/duplicated_key.rs [hunk 1]
```diff
@@ -18,6 +19,41 @@ impl Fix for DuplicatedKeyFixer<'_> {
...
+        lines: &mut Vec<LineEntry>,
+    ) -> Option<usize> {
+        let mut keys = HashSet::with_capacity(lines.len());
+        let mut is_disabled = false;
+
+        for line in lines {
+            if let Some(comment) = line.get_control_comment() {
+                if comment.checks.contains(&self.name) {
+                    is_disabled = comment.is_disabled();
+                }
```
[the remark this answers is not in the document]
Comment (evgeniy-r) [reply src/fixes/duplicated_key.rs:34]: We should not change `is_disabled` if it is an unrelated control comment.

File: src/fixes.rs [hunk 2]
```diff
@@ -49,28 +49,33 @@ fn fixlist() -> Vec<Box<dyn Fix>> {
...
 }
 
-pub fn run(warnings: &mut [Warning], lines: &mut Vec<LineEntry>) -> usize {
+pub fn run(warnings: &mut [Warning], lines: &mut Vec<LineEntry>, skip_checks: &[&str]) -> usize {
     if warnings.is_empty() {
         return 0;
     }
+    let mut fixes = fixlist();
+
+    // Skip fixes for checks in --skip argument (globally)
```
[the remark this answers is not in the document]
Comment (evgeniy-r) [reply src/fixes.rs:66]: OK, I will think about more suitable comment.

File: src/fixes/duplicated_key.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (mgrachev) [reply src/fixes/duplicated_key.rs:34]: Yes, you're right 👍

File: src/fixes.rs (hunk 2 above)
[the remark this answers is not in the document]
Comment (mgrachev) [reply src/fixes.rs:66]: I will do it later 🙂

**Review by mgrachev — APPROVED** [review]

**Comment by mgrachev** [comment]
@evgeniy-r Thanks a lot! 🔥

**Pull request merged**